library/src/main/scala/scala/scalajs/js/BigInt.scala

```diff
@@ -0,0 +1,188 @@
+package scala.scalajs.js
+
+import scala.scalajs.js
+import scala.scalajs.js.annotation.JSGlobal
+
+/**
+ * A built-in object that provides a way to represent whole numbers larger than
+ * 2 ^ 53 - 1, which is the largest number JavaScript can reliably represent
+ * with the Number primitive.
+ *
+ * BigInt can be used for arbitrarily large integers.
+ *
+ * @see https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/BigInt
+ */
```

Comment:
This will need a badge to make it clear it is not ES5.1 compliant.

Reply:
Added `@note`

Reply:
Added css-styed badge.